### PR TITLE
Fix guides index

### PR DESCRIPTION
**Pull request by qrkourier**

### CODE REVIEW

File: docusaurus/sidebars.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ const sidebars = {
   docsSidebar: [
     {
       type: 'html',
-      value: 'LEARN', // The HTML to be rendered
+      value: '<a href="/docs/learn/introduction/">LEARN', // The HTML to be rendered
```
Comment (dovholuknf) [inline docusaurus/sidebars.js]: this is improper html. this 'works'?

Comment (dovholuknf) [reply docusaurus/sidebars.js]: If you're going with the `<a>` approach -- i dunno if we should, you need to style the link to look the same. notice the REFERENCE difference

image

Comment (qrkourier) [reply docusaurus/sidebars.js]: Forgot the closing tag.

**Comment by qrkourier** [comment]
@dovholuknf this PR fixes the sidebar heading links and the misplaced index page for the Guides section

**Review by dovholuknf — DISMISSED** [review]

**Review by dovholuknf — APPROVED** [review]

**Pull request merged**